org.opent2t.sample.hub.superpopular/com.nest.hub/js/thingTranslator.js

```diff
@@ -0,0 +1,158 @@
+/* jshint esversion: 6 */
+/* jshint node: true */
+/* jshint sub:true */
+// This code uses ES2015 syntax that requires at least Node.js v4.
+// For Node.js ES2015 support details, reference http://node.green/
+
+"use strict";
+var request = require('request-promise');
+
+/**
+* This translator class implements the "Hub" interface.
+*/
+class Translator {
+    constructor(accessToken) {
+        this._accessToken = accessToken;
+
+        this._baseUrl = "https://developer-api.nest.com";
+        this._devicesPath = '/devices';
+
+        this._name = "Nest Hub"; // TODO: Can be pulled from OpenT2T global constants. This information is not available, at least, on wink hub.
```

Comment:
And in several other comments)

---
In reply to: [78475948](https://github.com/openT2T/translators/pull/92#discussion_r78475948) [](ancestors = 78475948)